### PR TITLE
Portfolio and Getting Hired: Added some resources 

### LINKED ISSUE #25822: <Getting Hired: What You Can Do To Prepare>: <More actionable ways to Get Good >
### Describe your suggestion

The "Get Good" subsection provides great tips, but I think it would be beneficial to mention more specific ways learners can improve and learn their skills. I've recently read some articles written by people who landed jobs as a result of [participating in the 100 Days of Code challenge](https://www.freecodecamp.org/news/how-freecodecamp-and-100daysofcode-helped-me-get-hired-in-under-a-year/) and [contributing to open source projects](https://dev.to/opensauced/how-i-got-hired-contributing-to-open-source-projects-546i), and I believe this can be a great way to emphasize the importance of getting good. 

##Update
Through other people’s suggestions, I will add the open source suggestion to the Portfolio section and raise a related issue about adding information on 100 Days of Code to the beginning section 

### Path

Foundations

### Lesson Url

https://www.theodinproject.com/lessons/node-path-getting-hired-what-you-can-do-to-prepare

### Checks

- [X] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/.github/blob/main/CONTRIBUTING.md)
- [X] Would you like to work on this issue?

### (Optional) Discord Name

Chrissy

### (Optional) Additional Comments

_No response_

### PR DESCRIPTION
**Pull request by CBID2**
## Because
Adding these two resources would help students stay motivated in their journey. 

## This PR
- References two articles about 100 Days of Code and how making open source contributions can be helpful in searching for developer roles. 

## Issue

Closes #25822 

## Pull Request Requirements

-   [X] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/.github/blob/main/CONTRIBUTING.md)
-   [X] The title of this PR follows the `location of change: brief description of change` format, e.g. `Intro to HTML and CSS lesson: Fix link text`
-   [X] The `Because` section summarizes the reason for this PR
-   [X] The `This PR` section has a bullet point list describing the changes in this PR
-   [X] If this PR addresses an open issue, it is linked in the `Issue` section
-   [X] If any lesson files are included in this PR, they have been previewed with the [Markdown preview tool](https://www.theodinproject.com/lessons/preview) to ensure it is formatted correctly
-   [X] If any lesson files are included in this PR, they follow the [Layout Style Guide](https://github.com/TheOdinProject/curriculum/blob/main/LAYOUT_STYLE_GUIDE.md)

### CODE REVIEW

File: foundations/introduction/motivation_and_mindset.md [hunk 1]
```diff
@@ -58,7 +58,7 @@ You can practice this method of learning by helping others in our community.
 *   To learn more about the best ways to learn, [learning how to learn on Coursera](https://www.coursera.org/learn/learning-how-to-learn) is highly recommended.
 
 *   The Ruby Rogues have a [podcast on How to Learn](https://topenddevs.com/podcasts/ruby-rogues/episodes/131-rr-how-to-learn), which should be motivational and useful to you, so check it out for some useful thoughts on learning.
-
+*   100 Days of Code is a challenge that aspiring developers often use to start their coding journey, so consider checking out [its website](https://www.100daysofcode.com/) if you are looking for another way to stay motivated and inspire others.
```
Comment (rlmoser99) [inline foundations/introduction/motivation_and_mindset.md:61]: This is missing a new line above and below this bullet point.

The heading will not render correctly without this new line. Please make sure to use the Lesson Preview Tool to catch things like this.
https://www.theodinproject.com/lessons/preview

File: getting_hired/preparing_for_job_search/project_portfolio.md [hunk 2]
```diff
@@ -4,7 +4,7 @@ You don't need a personal website right off the bat but it's good to have and it
 
 For this project, it's best for you to identify the personal websites of several people you admire and see what you think they've done right.  In general, it's best to keep your site simple, clean and to the point rather than bloated and full of "look what I can do!" features.  Let your portfolio of work speak for itself.
 
-* [Why Every Job Seeker Should Have a Personal Website](http://www.forbes.com/sites/jacquelynsmith/2013/04/26/why-every-job-seeker-should-have-a-personal-website-and-what-it-should-include/)
+* [Why Every Job Seeker Should Have a Personal Website](http://www.forbes.com/sites/jacquelynsmith/2013/04/26/why-every-job-seeker-should-have-a-personal-website-and-what-it-should-include/)
```
Comment (rlmoser99) [inline getting_hired/preparing_for_job_search/project_portfolio.md:7]: It looks like you added white space to the end of this line. Can you please remove it.

File: getting_hired/preparing_for_job_search/project_portfolio.md [hunk 3]
```diff
@@ -13,8 +13,8 @@ Don't underestimate the importance of high quality documentation on your project
 Remember to write some documentation for both technical and non-technical audience.  A potential employer should be able to follow your documentation, regardless of their technical background.
 
 * [A Beginner’s Guide to Writing Documentation](http://www.writethedocs.org/guide/writing/beginners-guide-to-docs/)
-
-### Show Designs
+* If you're looking for other projects to present in your portfolio, consider participating in the open source community. It's a great way to practice the skills you've developed from our curriculum and gain real-world experience. Check out [this article](https://dev.to/opensauced/how-i-got-hired-contributing-to-open-source-projects-546i) to learn how open source can help you in your job search. 
+
```
Comment (rlmoser99) [inline getting_hired/preparing_for_job_search/project_portfolio.md:17]: Can you explain why you removed the "Show Designs" heading?

In addition, can you explain why you put this in the portfolio project lesson, as opposed to the "What You Can Do to Prepare" lesson that you pointed to when you made the issue?

**Review by rlmoser99 — CHANGES_REQUESTED** [review]
Thanks for taking the time to make this PR. I've left a few questions and change requests.

**Comment by CBID2** [comment]
Hi @rlmoser99. I'm going to do a new version of this PR due to merge conflicts.

File: getting_hired/preparing_for_job_search/project_portfolio.md (hunk 3 above)
Comment (CBID2) [reply getting_hired/preparing_for_job_search/project_portfolio.md:17]: > Can you explain why you removed the "Show Designs" heading?
> 
> In addition, can you explain why you put this in the portfolio project lesson, as opposed to the "What You Can Do to Prepare" lesson that you pointed to when you made the issue?

Hi @rlmoser99! :) Here's a new version of my PR: https://github.com/TheOdinProject/curriculum/pull/25970

**Pull request closed without merging**